packages/frontend/src/ui/pool-remote-logo/index.tsx

```diff
@@ -14,14 +14,16 @@ interface PoolRemoteLogoProps {
 
 export function PoolRemoteLogo({
     loading,
-    size = "md",
+    size,
     chain,
     token0,
     token1,
     className,
 }: PoolRemoteLogoProps) {
     return (
-        <div className={`${styles.root} ${styles[size]} ${className?.root}`}>
+        <div
+            className={`${styles.root} ${size ? styles[size] : styles.selfAdjust} ${className?.root}`}
```

Comment:
I think we can use the `classNames` utility here to conditionally apply the size.; for example https://github.com/metrom-xyz/monorepo/blob/develop/packages/frontend/src/ui/date-picker/index.tsx#L126.

Reply:
Also I think the same thing applies https://github.com/metrom-xyz/monorepo/pull/163#discussion_r1746693085.